docs/src/content/docs/getting-started/concepts.mdx

```diff
@@ -13,12 +13,17 @@ To effectively use Terrateam, it's important to understand its key concepts and
 
 At the heart of Terrateam is the concept of a Dirspace. A Dirspace represents a unique combination of a repository, a directory, and a workspace. It is denoted as a tuple `(repository, directory, workspace)` or simply `(directory, workspace)` when the repository is implied.
```

Comment:
Dirspaces are introduced abstractly; provide a real-world example or use case before delving into the technical details.

Reply:
> Dirspaces are introduced abstractly; provide a real-world example or use case before delving into the technical details.

```suggestion
At the heart of Terrateam is the concept of a Dirspace. A Dirspace is a unique combination of a repository, directory, and workspace. It is denoted as a tuple `(repository, directory, workspace)` or simply `(directory, workspace)` when the repository is implied.
```